Warnings

With `-Wall` the `generated_data` branch has a warning:

```
[build] src/chemcache/bases/def2-ecp.cpp:59:10: error: unused variable 'make_shell' [-Werror,-Wunused-variable]
[build]     auto make_shell = [&r0](auto pure, auto l, const doubles_t& cs,
[build]          ^
```

--- a/utils/data_management/generate_basis.py
+++ b/utils/data_management/generate_basis.py
@@ -422,7 +422,7 @@ def _parse_bases_nw(basis_set_filepaths: list, sym2Z: dict,
                     # Indicate that we are in a basis block and should parse
                     basis_block = True
                     is_pure = (bstart_match['shelltype'] == "SPHERICAL")
-        
+
         # Did we get anything out of the file?
         if not tmp_basis:
             print("no basis information parsed")

--- a/utils/data_management/generate_basis.py
+++ b/utils/data_management/generate_basis.py
@@ -422,6 +422,12 @@ def _parse_bases_nw(basis_set_filepaths: list, sym2Z: dict,
                     # Indicate that we are in a basis block and should parse
                     basis_block = True
                     is_pure = (bstart_match['shelltype'] == "SPHERICAL")
+        
+        # Did we get anythin out of the file?
+        if not tmp_basis:
+            print("no basis information parsed")
+            sys.stdout.flush()
+            continue
 
         # Process the data read from the file
         bases[basis_set] = {}

--- a/utils/data_management/generate_basis.py
+++ b/utils/data_management/generate_basis.py
@@ -423,7 +423,7 @@ def _parse_bases_nw(basis_set_filepaths: list, sym2Z: dict,
                     basis_block = True
                     is_pure = (bstart_match['shelltype'] == "SPHERICAL")
         
-        # Did we get anythin out of the file?
+        # Did we get anything out of the file?
         if not tmp_basis:
             print("no basis information parsed")
             sys.stdout.flush()

